methods.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+class DetailedProduct {
```

Comment:
delete dummy once real function is added

Reply:
Definitely :)